RELEASES

```diff
@@ -0,0 +1,13 @@
+# Releases
+
+A new release happens every 6 weeks:
+
+- 1st day on the first month of each quarter
+- 15th day of the second month of each quarter
+
+## v1.0.0 (2018-08-15)
+
+- Initial release
+- Tested with:
+  - Golang 1.10.3
+  - u-root v1.0.0
```

Comment:
Add known major bug about not being able to handle multiple files with same GUID in the same folder.

Reply:
Done!